### DISCUSSION TITLE
[docs] How to get the screen size

### DISCUSSION THREAD

**Discussion by mynane** [tauri-apps/tauri#4626] [Q&A] [answered]
How to get the screen size

**Comment by FabianLars** [comment, marked as answer]
In javascript use this <https://tauri.app/v1/api/js/modules/window#currentmonitor> to get the monitor and `monitor.size` to get its size.
In Rust: basically the same starting with <https://docs.rs/tauri/latest/tauri/window/struct.Window.html#method.current_monitor>
